docs/en/data_types/datetime.md

```diff
@@ -1,45 +1,54 @@
+A moment of time is stored as Unix timestamp, independently of time zones and daylight savings. Additionally `DateTime` can store time zone, that affects how `DateTime` values are displayed in text format and how input strings are parsed for storage. You can find the list of supported time zones in the [IANA Time Zone Database](https://www.iana.org/time-zones).
 
-You can explicitly set a time zone for `DateTime`-type column when creating a table. If time zone isn't set, ClickHouse uses the value of the [timezone](../operations/server_settings/settings.md#server_settings-timezone) server configuration parameter or the operating system settings at the moment of the ClickHouse server start. 
+You can explicitly set a time zone for `DateTime`-type columns when creating a table. If the time zone isn't set, ClickHouse uses the value of the [timezone](../operations/server_settings/settings.md#server_settings-timezone) parameter in the server settings or the operating system settings at the moment of the ClickHouse server start.
 
-The [clickhouse-client](../interfaces/cli.md) applies the server time zone by default if a time zone isn't explicitly defined when initializing the data type. To use the client time zone, run it with the `--use_client_time_zone` parameter.
+The [clickhouse-client](../interfaces/cli.md) applies the server time zone by default if a time zone isn't explicitly set when initializing the data type. To use the client time zone, run `clickhouse-client` with the `--use_client_time_zone` parameter.
 
-ClickHouse outputs values in the `YYYY-MM-DD hh:mm:ss` text format by default. You can change the format with the [formatDateTime](../query_language/functions/date_time_functions.md#formatdatetime) function.
+ClickHouse outputs values in `YYYY-MM-DD hh:mm:ss` text format by default. You can change the format with the [formatDateTime](../query_language/functions/date_time_functions.md#formatdatetime) function.
 
-When inserting data into ClickHouse, you can use different formats of date and time strings, depending on the [date_time_input_format](../operations/settings/settings.md#settings-date_time_input_format) setting value.
+When inserting data into ClickHouse, you can use different formats of date and time strings, depending on the value of the [date_time_input_format](../operations/settings/settings.md#settings-date_time_input_format) setting.
 
 ## Examples
 
-Creating a table with a `DateTime`-type column:
+**1.** Creating a table with a `DateTime`-type column and inserting data into it:
 
 ```sql
-CREATE TABLE dt(
-    timestamp DateTime('Europe/Moscow')
-)
+CREATE TABLE dt( timestamp DateTime('Europe/Moscow') ) ENGINE TinyLog
+```
+```sql
+INSERT INTO dt Values (1546300800), ('2019-01-01 00:00:00')
+```
+```sql
+SELECT * FROM dt
+```
+```text
+┌───────────timestamp─┐
+│ 2019-01-01 03:00:00 │
+│ 2019-01-01 00:00:00 │
+└─────────────────────┘
```

Comment:
I would add some ID column
INSERT INTO dt Values (1, 1546300800), (2, '2019-01-01 00:00:00');

and example how to select it with a condition

select * from dt where timestamp = toDateTime('2019-01-01 00:00:00', 'Europe/Moscow')

Reply:
Thank you a lot, I'll add it.